requirements.txt

```diff
@@ -2,4 +2,5 @@ cython
 opencv-python
 pycocotools
 matplotlib
-sparseml[torchvision] >= 0.7.0
+sparseml
+torchvision<=0.8.2
```

Comment:
where is the torch requirement coming in for this?

Reply:
`torchvision 0.8.2` relies on the correct `torch` version needed; and installs that in turn